### PR TITLE
[react-intl] - POC i18n

### PR DESCRIPTION
**Pull request by Fuzznimp**
## Description

Here is my first take on implementing `i18n`. I did not fix the tests for now as it is not useful to discuss the implementation.

~I also implemented a `React.Context` at first, but it's (once again) not useful. I did not removed it as there are clear issue with fetching the locale client side.~ Nvm, after a discussion with fenn, we decided that, as a first implementation, we won't implement the storing in DB/context. Instead, we'll use the `NEXT_LOCALE` cookie, alongside the `ACCEPTED_LANGUAGE` header entry.

## Type of change

- [ ] **Chore** (non-breaking change which refactors / improves the existing code base).
- [ ] **Bug fix** (non-breaking change which fixes an issue).
- [x] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## List of added dependencies

"react-intl": "5.20.1"

## Screenshots

The preview app will do a better job :).

## Checklist

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

**Comment by frieda-ewlines** [comment]
Task linked: [CU-m36f49 [POC] - I18n](https://app.clickup.com/t/m36f49)

File: content/locales/fr.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    buttonLabel: "Accéder à mon compte",
+  },
+  "/account": {
+    breadcrumb: "Bienvenue sur votre compte",
+    createProfile:
+      "Ajoutez des informations de base sur vous, comme votre nom ou vos adresses.",
+    updatedProfile:
+      "Mettez à jour les informations de base vous concernant, comme votre nom ou vos adresses.",
+    logins:
+      "Gérez vos options de connexion, y compris les courriels, les numéros de téléphone et les comptes sociales",
```
Comment (Oluane) [inline content/locales/fr.ts:13]:
```suggestion
      "Gérez vos options de connexion, y compris les courriels, les numéros de téléphone et les comptes sociaux",
```

File: content/locales/fr.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    title: "Identifiants",
+    breadcrumb: "Vos e-mails, téléphones et identifiants sociaux",
+    emailTitle: "Adresses e-mail",
+    emailNoIdentityMessage: "Aucun e-mail n’a encore été ajouté.",
+    emailAddNewIdentityMessage: "Ajouter une nouvelle adresse e-mail",
+    phoneTitle: "Numéros de téléphone",
+    phoneNoIdentityMessage: "Aucun numéro de téléphone ajouté.",
+    phoneAddNewIdentityMessage: "Ajouter un nouveau numéro de téléphone",
+    socialTitle: "Comptes sociaux",
+    socialNoIdentityMessage: "Aucun compte sociale n’a encore été ajoutée",
```
Comment (Oluane) [inline content/locales/fr.ts:27]:
```suggestion
    socialNoIdentityMessage: "Aucun compte social n’a encore été ajouté",
```

**Review by Meyclem — CHANGES_REQUESTED** [review]

File: src/components/pages/locale/locale.tsx [hunk 3]
```diff
@@ -44,7 +44,7 @@ const Locale: React.FC = () => {
                 return;
               }
 
-              router && router.push("/account");
+              router && router.push("/account", undefined, { locale });
```
[the remark this answers is not in the document]
Comment (Fuzznimp) [reply src/components/pages/locale/locale.tsx:47]: It is an alias for the URL (name `as`). They did not overload the function, so you need to pass `URL | undefined` if you want to pass the options object.

File: content/locales/fr.ts [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    setBreadcrumb: "Mot de passe | créer",
+    updateBreadcrumb: "Mot de passe | mettre à jour",
+    set: "créer",
+    update: "mettre à jour",
+  },
+  "/account/security/sudo": {
+    title: "Sécurité",
+    warning:
+      "Vous avez besoin d’une authentification à double facteur pour accéder à cette page",
+    info: "Choisissez un moyen de contact ci-dessous que nous enverrons un code de validation à:",
```
[the remark this answers is not in the document]
Comment (Fuzznimp) [reply content/locales/fr.ts:46]: Omg 🤣

**Review by Meyclem — DISMISSED** [review]

**Review by Oluane — APPROVED** [review]

**Pull request merged**